### DISCUSSION TITLE
VDH no longer "sees" HD members-only videos on YouTube

### DISCUSSION THREAD

**Discussion by soundsgoodtome** [aclap-dev/video-downloadhelper#157] [Help and Q&A]
### Your question

Just a few days ago, I subscribed to a YouTube channel that offers advanced tutorials for a fee. I used VDH to capture a few of them for personal use after the subscription eventually ends. A couple of them downloaded with a watermark, so I even purchased a VDH license.

Now—four days later—VDH no longer even "sees" the full-sized members-only videos. Instead of showing me the various resolutions (sizes) of the video that can be downloaded, I'm shown only one: 256x144, which is of course ridiculous. (Screenshot attached.)

This change took place within the last 24-48 hours, so I assume it's related to the new 8.2 update about which others are reporting issues. It's frustrating to have finally paid for a license and then for the software to immediately break. I assume a fix is required by the developer for this? That said, if I'm overlooking something, and if there's a setting that can rectify this, I'd appreciate any suggestions.
 
<img width="517" alt="Screenshot 2024-02-17 at 4 13 07 PM" src="https://github.com/aclap-dev/video-downloadhelper/assets/1616664/c87ed4a2-d4b4-41cf-a15e-6d2a852680ce">

### Addon & CoApp version and details

Video DownloadHelper
Version 8.2.0.23 (stable)
Browser locale: en-US
Production build
Built on 2024-02-14
Build options: browser=firefox, linuxlic=false, noyt=false, target=mozilla

Found companion app: VdhCoApp 2.0.10
Companion app binary: /Applications/net.downloadhelper.coapp.app/Contents/MacOS/vdhcoapp

Firefox 122.0.1
macOS 14.4

**Comment by jcfrog** [comment]
Hi,
The 8.2.0.23 has several issues that should be fixed very soon.

**Comment by soundsgoodtome** [comment]
Thanks!

On Feb 17, 2024, at 11:41 PM, jcfrog ***@***.***> wrote:﻿
Hi,
The 8.2.0.23 has several issues that should be fixed very soon.

**Comment by paulrouget** [comment]
Also, try this:

In the settings, click More, then Video Qualities, and uncheck "Ignore low quality videos".

Tell me if this improves the situation.

Thanks!

**Comment by soundsgoodtome** [comment]
Thanks for the follow-up. Unfortunately no, unchecking that option does not improve the situation.

**Reply by GrampaWildWilly** [reply]
I also hit one something like this yesterday.  It wasn't a paid video, just something you needed to log in for.  Let me see if I can find it again . . .

https://www.youtube.com/watch?v=wfLSzt04F6o

I ended up messing around with the Firefox Network Monitor & squeezing the URLs of the video & audio tracks out of the json with the word player in its name.  That let me use the Firefox Save Video/Audio As function, which worked just fine.  So the fact that I could manually download this should mean that VDH will eventually be able to handle this.  It has been a problem of long standing that if you have to log into YouTube, either with a paid ID like our original poster here or with a free one like me, VDH does not work.  The VDH menu simply remains empty or shows a bunch of small items that can't possibly be the content you want.

**Comment by jawhelan** [comment]
I'm still not able to download HD on memeber only on youtobe.

version: 9.1.0.42
target: mozilla
channel: stable
lang: en-US
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"accepted":true,"email":"<removed>","key":"bc6942fbbd314f7c"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:129.0) Gecko/20100101 Firefox/129.0
{
    "valid_license_message_has_been_displayed": true,
    "used_history_button": true,
    "view_options": {
        "all_tabs": true,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": true,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "use_sidebar": false,
    "open_count_store": 5,
    "record_download_history": true
}
